pkg/gen/input/workflows/internal/file/json_schema_validation.yaml.template

```diff
@@ -0,0 +1,61 @@
+{{{{ .Header }}}}
+
+name: JSON schema validation
+on:
+  pull_request: 
+    branches:
+      - master
+      - main
+    paths:
+      - 'helm/**/values.yaml'
+      - 'helm/**/values.schema.json'
+
+jobs:
+  validate:
+    name: Validate values.schema.json with schemalint
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "^1.19"
+
+      - name: Install schemalint
+        run: |
+          go install github.com/giantswarm/schemalint@latest
```

Comment:
IMO we shouldn't install `latest` here to not break peoples builds in case the latest version of schemalint introduces some breaking changes.

Also other workflows use https://github.com/giantswarm/install-binary-action to install pre-built binaries in GitHub actions.

Reply:
Nice idea, I'm looking into it :+1:

Reply:
I created a composite action in the schemalint repository and now use it here.
I've also requested your review on [the PR](https://github.com/giantswarm/schemalint/pull/57) that adds the action to the schemalint repository. I would appreciate if you could take a look at it!